packages/gestalt/src/ExpandableModule.jsdom.test.js

```diff
@@ -0,0 +1,80 @@
+// @flow strict
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import ExpandableModule from './ExpandableModule.js';
+
+describe('ExpandableModule', () => {
+  const props = {
+    items: [
+      {
+        title: 'Title1',
+        icon: 'lock',
+        iconAccessibilityLabel: 'lock icon label',
+        accessibilityLabel: 'tap to expand or collapse the module',
+        accessibilityControls: 'ChildId1',
```

Comment:
@galaxy0101 instead of having to define `accessibilityControls` and `accessibilityLabel` on each item, could we do the following:

```
<ExpandableModule
  id="expandableUniqueID"
  accessibilityExpandLabel="Tap to expand the module"
  accessibilityCollapseLabel="Tap to collapse the module"
  items={[
        title: 'Title1',
        icon: 'lock',
        iconAccessibilityLabel: 'lock icon label',
        summary: ['summary1'],
        children: 'Children1',
  ]}
```

* `id`, `accessibilityExpandLabel` and `accessibilityCollapseLabel` are all required
* We no longer have to set a separate `accessibilityLabel` for each item
* We can dynamically set the `accessibilityControls`, based on the index of the item

Reply:
```
accessibilityControls=`${id}-${index}`
```

Reply:
I wonder about exposing accessibilityControls  or not. Do screen readers read the id? Does that id provide any information?  I thought about this solution but I wasn't sure if templated ids would not provide enough context. I guess their only function is to link the dependent components, so that might be it.

Reply:
Updated